pg_search/src/index/search.rs

```diff
@@ -403,57 +389,17 @@ impl<T> From<PoisonError<T>> for SearchIndexError {
 
 #[cfg(test)]
 mod tests {
-    use crate::{fixtures::*, schema::SearchConfig};
-    use rstest::*;
-    use tantivy::schema::OwnedValue;
-
     use super::SearchIndex;
+    use crate::{
+        fixtures::{mock_dir, MockWriterDirectory},
+        writer::SearchFs,
+    };
+    use rstest::*;
 
     /// Expected to panic because no index has been created in the directory.
     #[rstest]
     #[should_panic]
     fn test_index_from_disk_panics(mock_dir: MockWriterDirectory) {
         mock_dir.load_index::<SearchIndex>().unwrap();
     }
-
-    #[rstest]
-    fn test_chinese_compatible_tokenizer(mut chinese_index: MockSearchIndex) {
-        let client = TestClient::new_arc();
-
```

Comment:
I removed this "unit" test because it is incompatible with the `SearchIndex` now being created in the background writer. We get all kinds of `pgrx` memory errors at runtime.

The `chinese_compatible` functionality is adequately covered in our integrations tests, so this really isn't needed.